frame/multisig/src/lib.rs

```diff
@@ -378,9 +400,12 @@ pub mod pallet {
 				maybe_timepoint,
 				CallOrHash::Call(*call),
 				max_weight,
+				None,
 			)
 		}
 
+		/// DEPRECATED: Use the `approve_as_multi` extrinsic instead.
+		///
```

Comment:
maybe add a note here mentioning this always has an expiry of `None` where the other allows customising it?